demo/tests/integration_test.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018-2020 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+// You should run `cargo test` with `--test-threads=1`, if you use `demo` feature.
+#![cfg(feature = "demo")]
+
+use std::thread::sleep;
+use std::time::Duration;
+
+#[test]
+fn run() {
+    let mut child = foundry_demo::run_node(4444);
+    sleep(Duration::from_secs(3));
+    child.kill().unwrap();
+    child.wait().unwrap();
+}
+
+#[actix_rt::test]
+async fn ping() {
+    let mut child = foundry_demo::run_node(5555);
+    sleep(Duration::from_secs(3));
+    let x = foundry_demo::request_query(5555, "ping", "aaaa", "aaaa").await;
+    assert_eq!(x, "Module not found: ping");
+    child.kill().unwrap();
+    child.wait().unwrap();
+}
```

Comment:
Will you run 4 Foundry instances in tests?
I think creating a binary is better than running 4 nodes in the test.
What is your plan?

Reply:
I just planned to call `foundry_demo::run_node()` multiple times. I don't understand what exactly `creating a binary` mean.

Reply:
What I mean is that creating a main function and call four `run_node` in the main function.
 I thought the demo will work more than a few tens of minutes. Running a test for more than few minutes looks awkward.

Reply:
Then how to deal with multiple tests?

Reply:
What are your scenarios?

It seems we have misunderstood each other.

I thought the demo will be like this:
1. Run 4 nodes and make them connect.
2. Send transactions periodically to the network and watch them.
I thought we would run Foundry nodes for days or weeks.

We need to make a consensus about what is our goal.
How about asking details for Kwangyul?

Reply:
I thought that we would run the demo for 2 or 3 mins. I'll ask him.

Reply:
@majecty What is the conclusion considering KwangYul's comment?

Reply:
Let's make a binary that runs 4 nodes.
It will be easy to app developers to run the network and easy to kill the netwok by killing the process.